htdocs/core/class/hookmanager.class.php

```diff
@@ -130,24 +130,25 @@ function executeHooks($method, $parameters=false, &$object='', &$action='')
 
         // Define type of hook ('output' or 'addreplace'. 'returnvalue' is deprecated).
         $hooktype='output';
-        if (in_array(
-        	$method,
-        	array(
-        		'addMoreActionsButtons',
-		        'addStatisticLine',
-        	    'deleteFile',
-        		'doActions',
-        		'formCreateThirdpartyOptions',
-		        'formObjectOptions',
-		        'formattachOptions',
-		        'formBuilddocLineOptions',
-		        'moveUploadedFile',
-		        'pdf_writelinedesc',
-		        'paymentsupplierinvoices',
-		        'printSearchForm',
-        		'formatEvent'
-        		)
-        	)) $hooktype='addreplace';
+		if (in_array(
+			$method,
+			array(
+				'addMoreActionsButtons',
+				'addStatisticLine',
+				'deleteFile',
+				'doActions',
+				'formCreateThirdpartyOptions',
+				'formObjectOptions',
+				'formattachOptions',
+				'formBuilddocLineOptions',
+				'moveUploadedFile',
+				'pdf_writelinedesc',
+				'paymentsupplierinvoices',
+				'printAddress',
```

Comment:
Please note I just added this line but reindented the whole code block with tabs for consistency with Dolibarr coding style.